selfdrive/car/hyundai/interface.py

```diff
@@ -115,12 +115,12 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       if candidate in (CAR.IONIQ, CAR.IONIQ_EV_LTD, CAR.IONIQ_PHEV_2019):
         ret.minSteerSpeed = 32 * CV.MPH_TO_MS
     elif candidate == CAR.VELOSTER:
-      ret.mass = 3558. * CV.LB_TO_KG
+      ret.mass = 3558. * CV.LB_TO_KG + STD_CARGO_KG
```

Comment:
```suggestion
      ret.mass = 2917. * CV.LB_TO_KG + STD_CARGO_KG
```
also kinda close to gross (3700 lbs)

![image](https://github.com/commaai/openpilot/assets/25857203/9778f371-7f37-4e10-b050-ec0b3a3ca4d1)